test/formats.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+cd scripts
+python alternate_bulk_formats.py
+cd -
+if [ $(git diff alternate_formats/* | wc -l) -eq 0 ]; then
+  echo "No alternate format changes needed..."
+else
+  echo "Alternate formats need regenerated.  Run alternate_bulk_formats.py"
```

Comment:
The yaml/json deep comparison seems easy, thats a good idea.  The CSV part... since its not 1:1, should I just leave what is in alternate_bulk_formats alone, and separately create a json file for each yaml?  Or remove CSV from alternate_bulk_formats?